Makefile

```diff
@@ -90,7 +86,7 @@ vet:
 
 .PHONY: lint
 lint: verify.tidy
-	golangci-lint run ./...
+	golangci-lint run -v
```

Comment:
++ for added verbosity, and as far as I can tell the removal of `./...` should have a `nil` effect :+1: